src/main/scala/com/danielasfregola/twitter4s/entities/NewDM.scala

```diff
@@ -0,0 +1,5 @@
+package com.danielasfregola.twitter4s.entities
+
+private[twitter4s] final case class NewDM(event: NewEvent)
```

Comment:
Can we please rename this to `NewDirectMessageEvent` or something?

Reply:
ok